client/config/index.js

```diff
@@ -12,8 +12,4 @@ if ( 'undefined' === typeof window ||  ! window.config ) {
  * @property {Function} anyEnabled whether one of a set of properties is enabled
  * @property {Function} isEnabled whether a specific property is enabled
  */
-const config = window.config;
-config.isEnabled = window.isEnabled;
-config.anyEnabled = window.anyEnabled;
-
-export default config;
+export default require( '../../server/config' );
```

Comment:
This is mostly related to the way Node resolves module loading on client and server.

On server it scans `server` and then `client` server. On client is scans only `client`. I omitted `node_modules` which is the last place to check for module.

Config is a great example in this context because we load different content depending which part of code references it. We assume we never load server code to avoid cyclic dependencies and to make sure there is an easy way to look for dependencies. I would say it makes everything easier, because otherwise you could have code going back and forth between client and server folders.